Bottom: 18c89d887854c06d5365edff55525ef848db9e82
Top:    016cbbf35c4236dccf56097d91311ce00713e1a4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 02:53:15 +0000

Refresh of draw-c-support-for-coloring-ob

---

diff --git a/src/undo.h b/src/undo.h
index f415c3c..c52b051 100644
--- a/src/undo.h
+++ b/src/undo.h
@@ -33,7 +33,7 @@
 #include "global.h"
 
 #define DRAW_FLAGS	(RATFLAG | SELECTEDFLAG | SQUAREFLAG \
-			| HIDENAMEFLAG | HOLEFLAG | OCTAGONFLAG | FOUNDFLAG | CLEARLINEFLAG)
+			| HIDENAMEFLAG | HOLEFLAG | OCTAGONFLAG | CONNECTEDFLAG | FOUNDFLAG | CLEARLINEFLAG)
 
 											/* different layers */
